### ISSUE TITLE
UPX v3.95 can't recognize new header on AMD64 binary files compiled MS VS2019 and not pack it.

### ISSUE THREAD

**Issue by VictorVG** [upx/upx#320]
## What should have happened?

UPX v3.95 can't recognize new header on AMD64 binary files compiled MS VS2019 and not pack it.

If user try pack any x64 binary files compiled use MS VC++2019 then have error messages:

 `CantPackException: size of Load Configuration directory unexpected`

## How can we reproduce the issue?

1. for test try use dbgcore.dll v10.0.18362.1 (included in to Windows 10 SDK v10.0.18362.0, CMake x64 v3.16.0, SumatraPDF v3.2 Git-af4a9afc9 for AMD64 (for build needed only VC++2019).  
2. example:
```upx -9 cmake.exe
                       Ultimate Packer for eXecutables
                          Copyright (C) 1996 - 2018
UPX 3.95w       Markus Oberhumer, Laszlo Molnar & John Reiser   Aug 26th 2018

        File size         Ratio      Format      Name
   --------------------   ------   -----------   -----------
upx: cmake.exe: CantPackException: size of Load Configuration directory unexpected

Packed 1 file: 0 ok, 1 error.
```
## Please tell us details about your environment.

* UPX version used (`upx --version`): 3.95w
* Host Operating System and version: any version
* Host CPU architecture: x64
* Target Operating System and version: any version
* Target CPU architecture: x64

**Comment by bitraid** [comment]
This is fixed in devel branch. See #245.

**Comment by VictorVG** [comment]
@bitraid 
@jreiser 

Big thanks! And then a friend of mine asked if it is possible to compress SumatraPDF v3.2 Git-94084b94d x64 and it is going to MSVS2019 - he also discovered this phenomenon - x86 assembly is packing, and x64 causes an error message. I looked at other binaries compiled by these compilers and expressed the hypothesis described.

**P.S.**

It means that it was not in vain that we were taught at the institute (I graduated from Moscow Institute of Energy and Economics) on real industrial problems. I still gratefully recall my teachers, professors Larionov, Bessonov, Frolov, member of the cores of Petropolsky and Yevtikhiev, academician Voronov, although it has been more than thirty years since they taught us, but most importantly they taught us to think outside the box and look for approaches to unsolved problems. There are no unsolvable problems - just there are those problems for which we now lack knowledge, which means we need to sit down at books and devices and deal with what we met and then any problem that seems to us to be solved will be solved.
